examples/crypto/private_key_ecdsa.py

```
@@ -1,18 +1,21 @@
 """
+
 Example file: Working with ECDSA (secp256k1) PrivateKey using the PrivateKey class
-*WARNING* ECDSA seeds should not be printed or exposed in a real‑world scenario
+*WARNING* ECDSA seeds should not be printed or exposed in a real‑world scenario.
```

Ruff (0.14.14)

[warning] Docstring contains ambiguous `‑` (NON-BREAKING HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)